browser/brave_browser_process_impl.cc

```diff
@@ -116,11 +117,20 @@ brave_shields::AutoplayWhitelistService*
 BraveBrowserProcessImpl::autoplay_whitelist_service() {
   if (!autoplay_whitelist_service_) {
     autoplay_whitelist_service_ =
-        brave_shields::AutoplayWhitelistServiceFactory();
```

Comment:
nit: 4 spaces is ok here. I did a grep for =$ and then checked a few results and from what I could find continuations like that are 4 spaces.